Cython/Compiler/Main.py

```diff
@@ -660,12 +702,27 @@ def compile(source, options = None, full_module_name = None, **kwds):
     checking is requested, a CompilationResult is returned, otherwise a
     CompilationResultSet is returned.
     """
+    from ..Build.Cache import Cache
```

Comment:
This should go under `if options.cache` below because we don't need it otherwise.

Reply:
```suggestion
```